src/main.rs

```diff
@@ -85,17 +84,10 @@ enum Command {
     /// Sync a directory with S3
     #[structopt(name = "sync")]
     SyncCmd {
-        /// The direction of the sync: 'up' for local to remote, 'down' for remote to local
-        #[structopt(long, default_value = "up")]
-        direction: SyncDirection,
         #[structopt(long)]
-        bucket: String,
-        #[structopt(long)]
-        key: String,
-        /// The directory to use for up/down sync
+        source: String,
         #[structopt(long)]
-        directory: String,
-        /// Optional include glob pattern (see man 3 glob)
+        destination: String,
```

Comment:
Should be possible for the type here to be SyncParam, structopt should automatically pick up the FromStr instance.